workspace/busStop.ts

```diff
@@ -0,0 +1,31 @@
+const request = require('request')
```

Comment:
While this works, I'd much prefer we stick to typescript notation for imports (& everything really, thus my complaints about using `function` in Index).

Can you convert this (and all requires) into `import * as request from 'request'`?